apps/portal-front/schema.graphql

```diff
@@ -382,6 +388,19 @@ type CanEnrollResponse {
   isSameOrganization: Boolean
 }
 
+type EnrolledInstance {
+  isAllowed: Boolean!
+  organizationId: ID!
+}
+
+type CanUnenrollResponse {
+  instance: EnrolledInstance
```

Comment:
Simpler?

```suggestion
type CanUnenrollResponse {
  isAllowed: Boolean!
  organizationId: ID!
```

Reply:
Just put a bool instanceExists ? Or a composition Type as a result.
```
type CanUnenrollNotExistError {
  error: Boolean!  // => always true
}
```
or throw a JS error ?

Reply:
I chose the bool telling if the instance exists or not 👍